pkg/agent/multicast/mcast_controller.go

```diff
@@ -82,7 +84,8 @@ func (c *Controller) addGroupMemberStatus(e *mcastGroupEvent) {
 	status := &GroupMemberStatus{
 		group:          e.group,
 		lastIGMPReport: e.time,
-		localMembers:   sets.NewString(e.iface.InterfaceName),
+		localMembers:   map[string]time.Time{e.iface.InterfaceName: e.time},
+		ofGroupID:      c.v4GroupAllocator.Allocate(),
```

Comment:
where does this get Released?

Reply:
Good catch, I will call it when calling UninstallGroup